components/molecules/CollaborationCard/collaboration-card.tsx

```diff
@@ -0,0 +1,34 @@
+import { User } from "@supabase/supabase-js";
+import Avatar from "components/atoms/Avatar/avatar";
+import Button from "components/atoms/Button/button";
+import React from "react";
+
+export type CollaborationRequestObject = {
+  requestor: DbUser | undefined;
+  outreachMessage: string;
+};
+interface CollaborationCardProps extends CollaborationRequestObject {}
+const CollaborationCard = ({ requestor, outreachMessage }: CollaborationCardProps) => {
+  return (
+    <div className="bg-white flex flex-col gap-4 rounded-2xl p-4 border border-light-slate-6 max-w-[41.625rem]">
+      <div className="flex justify-between text-sm items-center ">
+        <div className="flex gap-2 text-sm items-center">
+          <Avatar
+            className="!rounded-none"
+            size="sm"
+            avatarURL="https://images.unsplash.com/photo-1534528741775-53994a69daeb?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1064&q=80"
+          />
+
+          <div>Microsoft</div>
```

Comment:
Does it have to be `Microsoft` all the time?

Reply:
This is just a placeholder until we are certain on what data will be coming from the backend 🍕

Reply:
There are two designs and the one with people should be the one we use. It should not be a faceless company, but rather a persons face.

cc @getaheaddev for clarification

Reply:
The current design show's the users/org avatar on the collaboration request card... I need some clarification when you say `two designs`.
Do we have another collaboration design aside this current implemented one?

Reply:
The current design show's the users/org avatar on the collaboration request card... I need some clarification when you say `two designs`. Do we have another collaboration design aside this current implemented one?